File: scienceCenterLukaFrontend/src/app/model/reviewing-dto.ts
import { Article } from './article';
import { Magazine } from './magazine';
import { OpinionAboutArticle } from './opinion-about-article';

export class ReviewingDto{

     article: Article;
	 magazine: Magazine;
     opinion: OpinionAboutArticle;
     
     editorOpinion: OpinionAboutArticle;

     insideMf: boolean;

    constructor(){}
}

File: scienceCenterLuka/src/main/java/com/project/dto/ReviewingDto.java
package com.project.dto;

import java.io.Serializable;
import java.util.List;

import com.project.model.OpinionAboutArticle;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ReviewingDto implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = -8111994278994400192L;
	
	private ArticleDto article;
	private MagazineDto magazine;
	private OpinionAboutArticle opinion;
	
	private List<OpinionAboutArticle> authorsMessages;

	
	private boolean insideMf;

}
